code/modules/admin/admin_verbs.dm

```diff
@@ -1016,6 +1016,24 @@
 
 	else if(istype(whom, /client))
 		recipient = whom
+		
+
+
+	if(irc)
```

Comment:
is this discord or irc

Reply:
The convention on /tg/ is to call anything external related IRC.

Reply:
do we have working irc code?

Reply:
Yes. This both works with Discord  and irc. It only sends a message to TGS which then decides which channel to use (either irc or discord.)